omero/developers/whatsnew.txt

```diff
@@ -47,51 +23,12 @@ The API's request operations :javadoc:`Chgrp
 <slice2html/omero/cmd/Chown.html>`, :javadoc:`Delete
 <slice2html/omero/cmd/Delete.html>`, and their superclass
 :javadoc:`GraphModify <slice2html/omero/cmd/GraphModify.html>`, are now
```

Comment:
"Chmod" and "Chmod2" should also be mentioned (similar URLs as for Chgrp) but awkwardly wasn't deprecated until 5.1.